src/background_remover.cc

```diff
@@ -306,7 +336,11 @@ void BackgroundRemover::maskBackground(cv::Mat &frame /* rgb */, const cv::Mat &
         cv::blur(mask, mask, cv::Size(blur_width, blur_width));
     }
 
-    maskImage.copyTo(frame, mask);
+    if (do_blend_layers) {
+      blendLayers(frame, maskImage, mask, frame);
```

Comment:
I’m assuming it’s cv::blur that sets pixel values to !255? If so, will `blendLayers` do anything different than `maskImage.copyTo` if `do_blur_mask` is false? Maybe add a warning that blending is only effective when also smoothing, or just move this `if` into the one above. (I assume `copyTo` is still quicker than `blendLayers` with a 255-or-0-weight.)

Reply:
It does _something_, although I don't know exactly what or why.

Reply:
Scaling the mask already creates values in between, so there is some effect even without blurring.